File: Problem 7/ans.cpp
#include<iostream>

using namespace std;

int prime(long long int x)
{
    for (int a=2;a<x;a++)
        if (!(x%a))
            return 0;
    return 1;
}
int main()
{
    long long int a=1,cnt=0;
    while(cnt!=10001)
    {
        a++;
        if (prime(a))
            cnt++;
    }
    cout << "10001st prime number: "<< a <<endl;
    return 0;
}

File: Problem 7/ans.py
def prime(x):
    for i in range(2,a):
        if not x%i:
            return 0
    return 1

cnt=0
a=1
while cnt!=10001:
    a+=1
    if prime(a):
        cnt+=1
print("10001st prime number: %d"%a)